pkg/util/apptainerconf/config.go

```diff
@@ -325,20 +345,22 @@ sessiondir max size = {{ .SessiondirMaxSize }}
 # DEFAULT: NULL
 # Only allow containers to be used that are located within an allowed path
 # prefix. If this configuration is undefined (commented or set to NULL),
-# containers will be allowed to run from anywhere on the file system. This
-# feature only applies when Apptainer is running in SUID mode and the user is
-# non-root.
+# containers will be allowed to run from anywhere on the file system.
+#
+# Only effective in setuid mode, with unprivileged user namespace creation disabled.
+# Ignored for the root user.
 #limit container paths = /scratch, /tmp, /global
 {{ range $index, $path := .LimitContainerPaths }}
 {{- if eq $index 0 }}limit container paths = {{ else }}, {{ end }}{{$path}}
 {{- end }}
 
 # ALLOW CONTAINER ${TYPE}: [BOOL]
 # DEFAULT: yes
-# This feature limits what kind of containers that Apptainer will allow
-# users to use (note this does not apply for root).  Note that some of the
-# same operations can be limited in setuid mode by the ALLOW SETUID-MOUNT
-# feature below; both types need to be "yes" to be allowed.
+# This feature limits what kind of containers that Singularity will allow
+# users to use.
+#
+# Only effective in setuid mode, with unprivileged user namespace creation disabled.
+# Ignored for the root user.
```

Comment:
Restore the previous sentence starting with `Note that some of the same operations...`.

Reply:
thanks! updated